usdmanager/parsers/usd.py

```diff
@@ -132,32 +133,43 @@ def parseMatch(self, match, linkPath, nativeAbsPath, fileInfo):
         # Propogate the extracted archive if this resolved file is in the same archive
         if self.extractedDir and fullPath.startswith(self.extractedDir + sep):
             queryParams.append("extractedDir=" + self.extractedDir)
-        
+
+        def pathForLink(path):
+            """Need three slashes before drive letter on Windows; this prepends one, so
+            with the usual two URL slashes we'll get the proper format."""
+            if os.name == 'nt' and re.match(r'^[a-zA-Z]:[/\\]', fullPath):
```

Comment:
We should probably pre-compile this regex at the top of this module and re-use it, since pathForLink will get called a _lot_ when large files are loaded.